### PR TITLE
[Qt] Use system defaults CA

### PR DESCRIPTION
**Pull request by tmpsantos**
The CA certificate we are bundling will expire in Aug/2018.

### CODE REVIEW

File: platform/qt/resources/common.qrc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <RCC>
     <qresource prefix="/">
+        <file alias="icon.png">../../../common/icon.png</file>
```
Comment (kkaefer) [inline platform/qt/resources/common.qrc]: unrelated change

Comment (tmpsantos) [inline platform/qt/resources/common.qrc]: Not really unrelated, this was on the same `qmapbox.qrc` file the ca-bundle used to be. I removed this `qmapbox.qrc` altogether and moved the icon to the `commom.qrc` shared by the examples, where the icon is used.

**Comment by tmpsantos** [comment]
For the record I tested this with `https://badssl.com/` and it works fine.
